packages/experiments/src/components/signals/Signals.scss

```diff
@@ -176,4 +175,8 @@ i.emailed {
 
 .bookmarkFilled {
   color: $ms-color-neutralSecondary;
-}
\ No newline at end of file
+}
+
+.needsRepublishing {
+  color: #d29200;
```

Comment:
None of the other Signal colors are hard-coded, should this one be? I imagine we'd want to pull from our standard palette.

Reply:
Good point. Updated file per comment.